cli/commands/role/create.go

```diff
@@ -0,0 +1,45 @@
+package role
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/sensu/sensu-go/cli"
+	"github.com/sensu/sensu-go/cli/commands/helpers"
+	"github.com/sensu/sensu-go/types"
+	"github.com/spf13/cobra"
+)
+
+// CreateCommand defines new command to create roles
+func CreateCommand(cli *cli.SensuCli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "create [NAME]",
+		Short:        "create new roles",
```

Comment:
Should that be "create a new role"?
Nitpicking, but it's to make sure people are not tricked into thinking this command can somehow add several roles at once.